robottelo/common/helpers.py

```diff
@@ -262,6 +265,7 @@ def configure_entities():
     configuration file is available.
 
     """
+    entity_mixins.CREATE_MISSING = True
```

Comment:
:tada: Pushing forward NailGun and maintaining backwards compatibility at the same time has been a fun challenge. I'm glad you like this.

SatelliteQE/nailgun#5 is closely related to this line of code.